kong/plugins/http-log/handler.lua

```diff
@@ -76,18 +76,23 @@ local function send_payload(self, conf, payload)
     end
   end
 
+  local headers = {
+    ["Host"] = parsed_url.host,
+    ["Content-Type"] = content_type,
+    ["Content-Length"] = #payload,
+    ["Authorization"] = parsed_url.userinfo and (
+      "Basic " .. encode_base64(parsed_url.userinfo)
```

Comment:
This is that I don’t fylly understand. It lets you set header name but it forcibly prefixes the value with Basic.